docs/extensions/sdk-composables.md

```diff
@@ -0,0 +1,132 @@
+---
+description: Learn about the extension SDK composables and how to utilize them when developing custom extensions.
+contributors: Esther Agbaje
+---
+
+# Extension SDK Composables
+
+The Extension Composables serve as a powerful toolkit to help developers create extensions in Directus.
+
+Rather than needing to rewrite logic from scratch, extension developers can leverage primitives like `useApi()` or
+`useStores()`, to abstract away complexity when building extensions.
```

Comment:
```suggestion
Rather than needing to rewrite logic from scratch, extension developers can leverage primitives like `useApi()` or
`useStores()`, to handle common complexities when building extensions.
```

Reply:
"Abstract away" works when spoken. Less in docs.